templates/lune/lune.amp.html

```diff
@@ -30,55 +30,53 @@
-                    layout="responsive">
-                      <!-- PNG Fallback -->
-                      <amp-img
-                        fallback
-                        class="lune-anim-scale-up z1"
-                        alt="{{alt}}"
-                        srcset="{{fallback_srcset}}"
-                        width="{{width}}"
-                        height="{{height}}"
-                        layout="responsive">
-                      </amp-img>
-                  </amp-img>
-                {{/product_image}}
-                <!-- Product name -->
-                <span class="lune-landing-title-middle lune-anim-move-up flex justify-between z1 absolute caps h3 md-h1" aria-hidden="true">{{#product_name_split}}<span>{{.}}</span>{{/product_name_split}}</span>
-                <!-- LU- -->
-                <span class="lune-landing-title-top lune-anim-move-up absolute inline-block z1 caps" aria-hidden="true">lu-</span>
-                <!-- -NE -->
-                <span class="lune-landing-title-bottom lune-anim-move-up absolute inline-block z1 caps" aria-hidden="true">-ne</span>
-              </figure>
-              <!-- Dictover CTA -->
-              <div class="absolute bottom-0 left-0 right-0 center z1 flex justify-center">
-                {{> partials/components/cta.snip.html}}
-              </div>
-            </div>
+            <figure class="lune-landing-content relative col-12 m0 flex-none">
+              <!-- Chair image -->
+              {{#product_image}}
+                <!-- WebP image -->
+                <amp-img
+                  class="lune-anim-scale-up z1"
+                  alt="{{alt}}"
+                  srcset="{{webp_srcset}}"
+                  width="{{width}}"
+                  height="{{height}}"
+                  layout="responsive">
+                    <!-- PNG Fallback -->
+                    <amp-img
+                      fallback
+                      class="lune-anim-scale-up z1"
+                      alt="{{alt}}"
+                      srcset="{{fallback_srcset}}"
+                      width="{{width}}"
+                      height="{{height}}"
+                      layout="responsive">
+                    </amp-img>
+                </amp-img>
+              {{/product_image}}
+              <!-- Product name -->
+              <span class="lune-landing-title-middle lune-anim-move-up flex justify-between z1 absolute caps h3 md-h1" aria-hidden="true">{{#product_name_split}}<span>{{.}}</span>{{/product_name_split}}</span>
+              <!-- LU- -->
+              <span class="lune-landing-title-top lune-anim-move-up absolute inline-block z1 caps" aria-hidden="true">lu-</span>
+              <!-- -NE -->
+              <span class="lune-landing-title-bottom lune-anim-move-up absolute inline-block z1 caps" aria-hidden="true">-ne</span>
+            </figure>
+            <!-- Dictover CTA -->
+            {{> partials/components/cta.snip.html}}
```

Comment:
Removed the wrapper around the CTA as with the new layout it's not necessary anymore